packages/Molecules/Form/Checkbox.vue

```diff
@@ -57,13 +56,20 @@ export default {
       this.$emit('change', event)
     }
   },
-  watch: {
-    checked (val) {
-      this.updateValue()
-      this.updateChange(val)
+  computed: {
+    checked: {
+      set (val) {
+        this.updateValue(val)
+        this.updateChange(val)
+      },
+      get () {
+        if (this.value === null || this.value === undefined) {
```

Comment:
Hi @dylankelly I am just wondering, instead of adding this `null` condition in watch for all form field, should we move this logic to the place set to `null` https://github.com/dpc-sdp/ripple/pull/224/files#diff-523d7114849b5c86e4a722370037e520R18 ? What about those Vue form generator fields? Are they all working without add this logic?

Reply:
the problem here is that each field (checkbox, date, etc) has a different default (initial blank value). Ie: checkbox is false, datepicker range is ['', '']. I feel like this is better placed in the fields themselves, otherwise RplClearsearch needs to know about the internal logic of all these different fields. Also this is not just for clear search, it allows you to initialise each field with null and it be handled properly. The default fields handle null as a blank value already.